odoo/addons/base/models/ir_model.py

```diff
@@ -1227,6 +1234,43 @@ def write(self, vals):
 
         return result
 
+    @api.model
+    def __update_reflected_field(self, table, field, value, selection_value):
```

Comment:
Method name is obscure.  What about
```suggestion
    def _substitute(self, value):
        """ Replace the selection 'self' to 'value' in the corresponding table. """
```
The table, column and old values are given by `self.env[self.field_id.model]._table`, `self.field_id.name` and `self.value`, respectively.

Reply:
I feel like simply `_substitute` is a bit too vague, maybe `_substitute_selection` is a good compromise?